src/components/NewMessageModal.vue

```diff
@@ -131,11 +132,79 @@ export default {
 				...data,
 				body: data.isHtml ? data.body.value : toPlain(data.body).value,
 			}
-			await sendMessage(data.account, dataForServer)
 
-			// Remove old draft envelope
-			this.$store.commit('removeEnvelope', { id: data.draftId })
-			this.$store.commit('removeMessage', { id: data.draftId })
+			const checkName = true
+			const checkAttachment = true
+
+			var warnings = ""
+			var lines = toPlain(data.body).value.split("\n")
+
+			if (checkName) {
+				const nameComponents = data.to.replaceAll('"', '').split(" ").filter(s => !s.includes("@"))
+				if (nameComponents.length > 0) {
+					var foundNameComponent = false
+					for (let component of nameComponents) {
+						if (lines[0].includes(component)) {
+							foundNameComponent = true
+							break
+						}
+					}
+
+					if (!foundNameComponent) {
+						warnings += "name in salutation"
+					}
+				}
+			}
+
+			if (checkAttachment && data.attachments.length === 0) {
+				var attachmentMentioned = false
+				for (let line of lines) {
+					if (line.startsWith(">") || line.startsWith("--")) {
+						break
+					}
+					if (line.includes('attachment') || line.includes('attached')) {
+						attachmentMentioned = true
+						break
+					}
+				}
+
+				if (attachmentMentioned) {
+					if (warnings !== "") {
+						warnings += ", "
+					}
+					warnings += "missing attachments"
```

Comment:
this needs translation

Reply:
@max65482  could you address this please?

Reply:
I will be looking at this PR again soon (with warnings only about missing attachments, not salutations for now).
(If you are referring to the translation: At this time I feel unable to address this.)